CRM/Utils/Date.php

```diff
@@ -2101,15 +2102,23 @@ public static function getDateFormat($formatType = NULL) {
   }
 
   /**
+   * Date formatting for imports where date format is specified.
+   *
+   * Note this is used for imports (only) because the importer can
+   * specify the format.
+   *
+   * Tests are in CRM_Utils_DateTest::testFormatDate
+   *
    * @param $date
+   *   Date string as entered.
    * @param $dateType
+   *   One of the constants like CRM_Core_Form_Date::DATE_yyyy_mm_dd.
    *
-   * @return null|string
+   * @return false|string
    */
   public static function formatDate($date, $dateType) {
-    $formattedDate = NULL;
     if (empty($date)) {
-      return $formattedDate;
+      return FALSE;
```

Comment:
Policy-wise I don't think you should be able to just change the signature of a longstanding public static function. And in fact null makes more sense to me as a return value here. Why false? Also see line 2144.

Reply:
hmm - ok - it is only used by import - but I'm not strongly pro 'false' - it just always seems like php functions generally do that when invalid

Reply:
Ok I thought I saw it elsewhere in universe but looks like really just civihr and I'm not sure what its status is. The use in sepa60 wouldn't make a difference. But still line 2144 here then.

Reply:
I updated back to NULL anyway